src/UI/Settings/General/GeneralViewTemplate.hbs

```diff
@@ -162,6 +162,68 @@
             </div>
         </div>
     </fieldset>
+    
+    <fieldset>
+        <legend>Proxy Settings</legend>
+        
+        <div class="form-group">
+            <label class="col-sm-3 control-label">Use Proxy</label>
+            
+            <div class="col-sm-4">
+                <select name="proxyEnabled" class="form-control x-proxy">
+                    <option value="off">Do not use Proxy</option>
+                    <option value="indexersOnly">Indexers Only</option>
+                    <option value="global">Use Proxy for everything</option>
+                </select>
+            </div>
+        </div>
+
+        <div class="x-proxy-settings">
+            <div class="form-group">
+                <label class="col-sm-3 control-label">Proxy Type</label>
+
+                <div class="col-sm-4">
+                    <select name="proxyType" class="form-control">
+                        <option value="http" selected="selected">HTTP(S)</option>
+                        <option value="socks4">Socks4</option>
+                        <option value="socks5">Socks5 (This option supports Tor)</option>
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-sm-3 control-label">Hostname</label>
+
+                <div class="col-sm-4">
+                    <input type="text" placeholder="localhost" name="proxyHostname" class="form-control"/>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-sm-3 control-label">Port</label>
+
+                <div class="col-sm-4">
+                    <input type="number" placeholder="9150" name="proxyPort" class="form-control"/>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-sm-3 control-label">Username</label>
+
+                <div class="col-sm-4">
+                    <input type="text" placeholder="optional" name="proxyUsername" class="form-control"/>
```

Comment:
Use help text instead of a placeholder.

https://github.com/Sonarr/Sonarr/blob/develop/src/UI/Settings/General/GeneralViewTemplate.hbs#L102-L104